addons/website_hr_recruitment/static/src/js/website_hr_applicant_form.js

```diff
@@ -37,36 +95,52 @@ publicWidget.registry.hrRecruitment = publicWidget.Widget.extend({
             $('#linkedin-message').removeClass('alert-warning').hide();
             $(ev.currentTarget).addClass('border-warning');
             $('#linkedin-message').text(_t("The value entered doesn't seems like a linkedin profile.")).addClass('alert-warning').show();
+        }
+
+        const job_id = $('#recruitment7').val();
+        const data = await rpc('/website_hr_recruitment/check_recent_application',
+            {
+                linkedin: linkedin,
+                job_id: job_id,
+            });
+        if (data.applied_same_job)  {
+            $('#linkedin-message').removeClass('alert-warning').hide();
+            $(ev.currentTarget).addClass('border-warning');
+            $('#linkedin-message').text(_t(data.message)).addClass('alert-warning').show();
+        } else if (data.applied_other_job)  {
+            $('#linkedin-message').removeClass('alert-warning').hide();
+            $(ev.currentTarget).addClass('border-warning');
+            $('#linkedin-message').text(_t(data.message)).addClass('alert-warning').show();
         } else {
             $(ev.currentTarget).removeClass('border-warning');
             $('#linkedin-message').removeClass('alert-warning').hide();
         }
     },
 
-    async _onFocusOutMail (ev) {
-        const email = $(ev.currentTarget).val();
-        if (!email) {
+    async _onFocusOutName (ev) {
```

Comment:
Any reason to have changed the order of the methods? I think they should be in the same order as the fields